app/src/main/java/com/codesoom/assignment/TaskHandler.java

```diff
@@ -0,0 +1,101 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.modles.Task;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class TaskHandler implements HttpHandler {
+    public static final String TO_JSON_FAIL = "Json conversion fail.";
+    public static final String TO_TASK_FAIL = "Task conversion fail.";
+
+    public static final String HANDLER_PATH = "/tasks";
+
+    private static final String EMPTY_STRING = "";
+    private static final String PATH_DELIMITER = "/";
+
+    private static final String HTTP_GET = "GET";
+    private static final String HTTP_POST = "POST";
+    private static final String HTTP_PUT = "PUT";
+    private static final String HTTP_PATCH = "PATCH";
+    private static final String HTTP_DELETE = "DELETE";
+
+    private final List<Task> tasks = new ArrayList<>();
+
+    private void sendResponse(
+            final HttpExchange exchange, final int statusCode, final String content
+    ) throws IOException {
+        final OutputStream outputStream = exchange.getResponseBody();
+        exchange.sendResponseHeaders(statusCode, content.getBytes().length);
+        outputStream.write(content.getBytes(StandardCharsets.UTF_8));
+        outputStream.close();
+    }
+
+    private void handleId(
+            final HttpExchange exchange, final String method, final String path
+    ) throws IOException {
+        if (HTTP_PUT.equals(method)) {
+            sendResponse(exchange, HttpURLConnection.HTTP_OK, HTTP_PUT + " " + path);
+        }
+
+        if (HTTP_PATCH.equals(method)) {
+            sendResponse(exchange, HttpURLConnection.HTTP_OK, HTTP_PATCH + " " + path);
+        }
```

Comment:
`return`을 이렇게 넣어주면 코드를 이해하기 더 쉽지 않을까요?

```suggestion
        if (HTTP_PUT.equals(method)) {
            sendResponse(exchange, HttpURLConnection.HTTP_OK, HTTP_PUT + " " + path);
            return;
        }

        if (HTTP_PATCH.equals(method)) {
            sendResponse(exchange, HttpURLConnection.HTTP_OK, HTTP_PATCH + " " + path);
            return;
        }
```

Reply:
네 맞습니다. 수정하겠습니다.